workspaces/orchestrator/plugins/orchestrator-backend/package.json

```diff
@@ -0,0 +1,112 @@
+      "@red-hat-developer-hub/backstage-plugin-orchestrator-backend",
+      "@red-hat-developer-hub/backstage-plugin-orchestrator-common"
+    ]
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./package.json": "./package.json"
+  },
+  "typesVersions": {
+    "*": {
+      "package.json": [
+        "package.json"
+      ]
+    }
+  },
+  "homepage": "https://red.ht/rhdh",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redhat-developer/rhdh-plugins",
+    "directory": "workspaces/orchestrator/plugins/orchestrator-backend"
+  },
+  "bugs": "https://github.com/redhat-developer/rhdh-plugins/issues",
+  "keywords": [
+    "support:tech-preview",
+    "lifecycle:active",
+    "backstage",
+    "plugin",
+    "orchestrator",
+    "workflows"
+  ],
+  "files": [
+    "app-config.yaml",
+    "dist",
+    "dist-dynamic/*.*",
+    "dist-dynamic/dist/**",
+    "static"
+  ],
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "tsc": "tsc",
+    "prettier:check": "prettier --ignore-unknown --check .",
+    "prettier:fix": "prettier --ignore-unknown --write .",
+    "lint:check": "backstage-cli package lint",
+    "lint:fix": "backstage-cli package lint --fix",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.25.0",
+    "@backstage/backend-defaults": "^0.5.2",
+    "@backstage/backend-plugin-api": "^1.0.1",
+    "@backstage/backend-tasks": "^0.6.1",
+    "@backstage/catalog-client": "^1.7.1",
+    "@backstage/errors": "^1.2.4",
+    "@backstage/integration": "^1.15.1",
+    "@backstage/plugin-catalog-node": "^1.13.1",
+    "@backstage/plugin-permission-common": "^0.8.1",
+    "@backstage/plugin-permission-node": "^0.8.4",
+    "@backstage/plugin-scaffolder-backend": "^1.26.2",
+    "@backstage/plugin-scaffolder-node": "^0.5.0",
+    "@red-hat-developer-hub/backstage-plugin-orchestrator-common": "^1.24.0",
+    "@urql/core": "^4.1.4",
+    "ajv-formats": "^2.1.1",
+    "cloudevents": "^8.0.0",
+    "express": "^4.18.2",
+    "express-promise-router": "^4.1.1",
+    "fs-extra": "^10.1.0",
+    "isomorphic-git": "^1.23.0",
+    "json-schema": "^0.4.0",
+    "moment": "^2.29.4",
+    "openapi-backend": "^5.10.5",
+    "yn": "^5.0.0"
+  },
+  "devDependencies": {
+    "@backstage/backend-test-utils": "1.0.2",
+    "@backstage/cli": "0.28.2",
+    "@janus-idp/backstage-plugin-audit-log-node": "^1.7.1",
+    "@janus-idp/backstage-plugin-rbac-common": "^1.12.1",
+    "@types/express": "4.17.21",
+    "@types/fs-extra": "11.0.4",
+    "@types/json-schema": "7.0.15",
+    "prettier": "3.3.3"
+  },
+  "peerDependencies": {
+    "@janus-idp/backstage-plugin-audit-log-node": "^1.7.1",
+    "@janus-idp/backstage-plugin-rbac-common": "^1.12.1"
```

Comment:
The rbac-common plugin was migrated to community-plugins, not sure if you'd want to use that instead: https://github.com/backstage/community-plugins/tree/main/workspaces/rbac/plugins/rbac-common

Reply:
Thanks! Fixed